tests/integrators/maxwell_curl.mac

```diff
@@ -0,0 +1,190 @@
+/*
+  Copyright (C) 2012 - 2015 by the deal.II authors
```

Comment:
@orlando-tian Did you add this file by mistake? I cannot make sense of it in the current deal.II testing framework (even though it looks interesting)...